### ISSUE TITLE
Linking error with Manjaro build of master branch

### ISSUE THREAD

**Issue by kilon** [Zylann/godot_voxel#387]
Seems the linker is not happy with building the tool on Manjaro with latest commit 008c38bb6be63858b484101a3b22f1d6b840599b
I tried also a previous commit that reports no build errors for CI 602ae1ea955a9165d728075c1f73944f9e5bac5f but i get the same error. Anyone else experience this ? 

I dont experience any compilers errors, I building with the latest commit of Godot 4 master branch

I paste the error bellow 

```
/usr/bin/ld: modules/libmodule_voxel.linuxbsd.opt.tools.64.a(voxel_blocky_model.linuxbsd.opt.tools.64.o): warning: relocation against `_ZN6zylann5voxel16VoxelBlockyModel9BakedData5Model12MAX_SURFACESE' in read-only section `.text'
/usr/bin/ld: modules/libmodule_voxel.linuxbsd.opt.tools.64.a(voxel_blocky_model.linuxbsd.opt.tools.64.o): in function `zylann::voxel::bake_mesh_geometry(zylann::voxel::VoxelBlockyModel&, zylann::voxel::VoxelBlockyModel::BakedData&, bool, zylann::voxel::VoxelBlockyModel::MaterialIndexer&)':
/home/kilon/coding/kyklops/modules/voxel/meshers/blocky/voxel_blocky_model.cpp:355: undefined reference to `zylann::voxel::VoxelBlockyModel::BakedData::Model::MAX_SURFACES'
/usr/bin/ld: warning: creating DT_TEXTREL in a PIE
collect2: error: ld returned 1 exit status
scons: *** [bin/godot.linuxbsd.opt.tools.64] Error 1
scons: building terminated because of errors.
[Time elapsed: 00:00:41.082]
mv: cannot stat './bin/godot.linuxbsd.opt.tools.64': No such file or directory
```

**Comment by MGilleronFJ** [comment]
First time I ever saw this in my life... no idea what the fix is supposed to be. `MAX_SURFACES` is a constant I defined the same way as many other constants in this module.

**Comment by duchainer** [comment]
Did the fix with @CarlosRios, I'll be opening a PR in a few.
Thanks a lot @MGilleronFJ for the suggestion, this fixed it on our Mac Builds :smile:

**Comment by Zylann** [comment]
Fix merged in f9690542b2defc6a8c60e3456d47609508b72a37
